backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -142,17 +152,23 @@ export class ContributionMessageResolver {
         lastName: contribution.user.lastName,
         email: contribution.user.emailContact.email,
         language: contribution.user.language,
-        senderFirstName: user.firstName,
-        senderLastName: user.lastName,
+        senderFirstName: moderator.firstName,
+        senderLastName: moderator.lastName,
         contributionMemo: contribution.memo,
       })
       await queryRunner.commitTransaction()
+      await EVENT_ADMIN_CONTRIBUTION_MESSAGE_CREATE(
+        { id: contribution.userId } as DbUser,
```

Comment:
So you hack the need to query it!?

Reply:
> So you hack the need to query it!?

Yes, I interpret this object ( `{ id: contribution.userId }` )  as DbUser. This just effects the compile time, not the runtime - but provides typesecurity, which I explicitly hack in this case, because I know this is the `userId` and I do not want to query the full User Object from the database explicitly.

This approach combines the benefits of typechecks and performance with the downside of a bit more code.